Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -13,6 +13,7 @@ var restObj = new restClient.RestCallbackClient(httpObj);
 
 var defaultAuthUrl = 'https://login.windows.net/';
 var azureApiVersion = 'api-version=2016-08-01';
+var defaultWebAppAvailabilityTimeout = 3000;
```

Comment:
Can you include the units in the name of variable for readability... defaultWebAppAvailabilityTimeoutInSecs or defaultWebAppAvailabilityTimeoutInMS